apicompat.go

```diff
@@ -217,15 +225,13 @@ func (c Checker) getDirsRecursive(base, rev, rel, prefix string) (dirs []string)
 func (c Checker) parseDir(rev, dir string) (pkg, error) {
 
 	// Use go/build to get the list of files relevant for a specific OS and ARCH
-
 	ctx := build.Default
 	ctx.ReadDir = func(dir string) ([]os.FileInfo, error) {
 		return c.vcs.ReadDir(rev, dir)
 	}
 	ctx.OpenFile = func(path string) (io.ReadCloser, error) {
 		return c.vcs.OpenFile(rev, path)
 	}
-	ctx.GOPATH = os.Getenv("GOPATH")
```

Comment:
I believe you, but that's pretty weird considering 
https://github.com/golang/go/blob/master/src/go/build/build.go#L265